3.6.0/postgres/docker-entrypoint.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
@gannebamm This file needs running permissions (chmod +x ), in order to be executed whithin the container.